spec/categorical_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe Daru::Vector do
+  context "initialize" do
+    let(:dv) { Daru::Vector.new [:a, 1, :a, 1, :c], type: :category }
+    subject { dv }
+    
+    it { is_expected.to be_a Daru::Vector }
+    its(:size) { is_expected.to eq 5 }
+    its(:type) { is_expected.to eq :category }
```

Comment:
My point regarding "`:numeric` not `:number`" was that `:categorical` would match the existing `:numeric` better, because both are adjectives. I think that maybe a reason why pandas devs have chosen dtype `category` in favor of `categorical` is that all the other dtypes are nouns, like `float`, `int` or `object` (of course, I have no idea of the true reasons the pandas devs had).

But I'm fine with it, if we keep using `:category` and not `:categorical`, since pandas is using it. Though I think that in the english language describing data as being of "categorical type" is more correct than "category type".